packages/ai-chat/src/common/chat-agents.ts

```diff
@@ -126,6 +127,7 @@ export interface ChatAgent extends Agent {
     locations: ChatAgentLocation[];
     iconClass?: string;
     invoke(request: MutableChatRequestModel, chatAgentService?: ChatAgentService): Promise<void>;
+    suggest?(context: ChatRequestModel | ChatSession): Promise<void>;
```

Comment:
```suggestion
```
Any reason this needs to be in the interface?

Reply:
Without that, we can't e.g. call `suggest` on the pinned agent when a session is created with one:

```
packages/ai-chat/src/common/chat-service.ts(193,40): error TS2339: Property 'suggest' does not exist on type 'ChatAgent'.
```

In general, if we want it known that making suggestions may be a capability of agents but isn't an obligatory one, we have to put it in the interface so that we can call it even if we don't know exactly which agent we're dealing with.

Reply:
But where do we currently call it from outside of the agent?

Reply:
We can add this in the future, if need I think

Reply:
So What I meant is, I would not expose this function now, because we do not need it anywhere